website/content/docs/enterprise/namespaces.mdx

```diff
@@ -82,7 +82,7 @@ namespace. This data includes the following:
 
 - Secret Engines
 - Auth Methods
-- Policies
+- Policies (including Password Policies)
```

Comment:
Can this be a separate line? Otherwise this could imply a relationship between policies and password policies when none exists

Reply:
Yep